app/src/core/document.ts

```diff
@@ -64,8 +64,8 @@ export async function deserializeDocument(zipBinary: Buffer): Promise<Document>
         console.log('namefile', file.name);
         const fileContents = await file.async('arraybuffer');
         console.log('filecontent', fileContents);
-        const decoded = await ctx.decodeAudioData(fileContents.slice(0));
-        return [basename(file.name), { fileContents, decoded }];
+        const objectUrl = URL.createObjectURL(new Blob([fileContents]));
```

Comment:
As dicussed (and if possible): use `file.async('blob')`, remove `fileContents` and `objectUrl` from Source and instead just pass a `Blob` object around

Reply:
Since chrome does not work according to spec, we cannot to this: https://bugs.chromium.org/p/chromium/issues/detail?id=506273